### PR TITLE
[WIP] Add executable permissions to some Android Tools 

### PR DESCRIPTION
**Pull request by nedo**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

**Comment by nedo** [comment]
Hi @estute some Android Tools need executable permissions, so I've created this PR. I'm testing it. I'll request review once done at my end.

**Comment by estute** [comment]
@nedo Thanks for the heads up. Is it ready for review now?

**Comment by nedo** [comment]
@estute it seems good after adding these permission, I think we should keep it active on save side(considering there may come some need)

**Comment by estute** [comment]
@nedo I am not sure I understand. Should I review the PR or do you want to continue working on it?

**Comment by nedo** [comment]
@estute I want to continue working on it. Thanks

**Comment by nedo** [comment]
@estute As discussed, please review this PR to check emulator implementations.

**Review by estute — APPROVED** [review]
Looks good to me. A couple of nits, but once you fix them, you should be good to merge! Let me know once this has merged, so that I can rebuild the android worker AMI.

File: playbooks/roles/android_sdk/tasks/main.yml [hunk 1]
```diff
@@ -83,4 +83,10 @@
 - name: Download Android System Image
   shell: "echo 'y' | {{ android_home }}/tools/bin/sdkmanager  {{ android_sys_image }} > /dev/null" 
   become: yes
-  become_user: "{{ android_user }}"
\ No newline at end of file
+  become_user: "{{ android_user }}"
+# Adding executable permissions
```
Comment (estute) [inline playbooks/roles/android_sdk/tasks/main.yml]: ^ no need for a comment like this when the task is named

Comment (nedo) [reply playbooks/roles/android_sdk/tasks/main.yml]: done

**Pull request merged**